ui/app/services/namespace.js

```diff
@@ -29,6 +30,11 @@ export default class NamespaceService extends Service {
     return this.path === ROOT_NAMESPACE;
   }
 
+  // the "root" namespace is "admin/" for HVD managed clusters accessing the UI
+  get inHvdAdminNamespace() {
+    return this.flags.isHvdManaged && this.path === 'admin/';
+  }
```

Comment:
Nice!

Reply:
what is they are in a nested path but on a hvdManaged cluster? ex: `admin/parent/child`?

Reply:
Some API endpoints are [restricted to the root/administrative namespaces](https://developer.hashicorp.com/vault/docs/enterprise/namespaces#restricted-api-paths)

This getter is specifically checking that the UI is in the administrative namespace, kind of like the "root" equivalent for HVD managed clusters. (Since `admin/` is the top-level namespace). So this getter is the HVD equivalent to `inRootNamespace` above. I opted to not combine the getters because there could be cases where we still explicitly want to check for the `root` namespace and _not_ the HVD administrative namespace. 

So we explicitly **don't** want this to be true for any child namespaces of `admin/`. Is there anything I can add do the comment to make this clearer?